intelmq/bots/collectors/rsync/collector_rsync.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from os import path, mkdir
+from subprocess import call
+from intelmq.lib.bot import CollectorBot
+from intelmq import VAR_STATE_PATH
+
+
+class RsyncCollectorBot(CollectorBot):
+    def init(self):
+        self.rsync_data_directory = path.join(VAR_STATE_PATH, "rsync_collector")
+        try:
+            mkdir(self.rsync_data_directory)
+        except FileExistsError:
+            pass
+
+    def process(self):
+        self.logger.info("Updating file {}.".format(self.parameters.file))
+        call(["rsync", path.join(self.parameters.rsync_path, self.parameters.file),
```

Comment:
The return value is not checked so. The command can fail without notice.

Reply:
Thanks for notice. Fixed.